control-plane-operator/hostedclusterconfigoperator/controllers/resources/alerts/assets/apiusage.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
```

Comment:
can we put this under a promtool check? see Makefile

Reply:
The api-usage rules need to be in the hosted cluster, but I believe promtool check will not install and reconcile objects inside hosted clusters, @enxebre please confirm.

Reply:
promtool is just tooling to prevent syntax errors in the rule yaml we'd want to run over the file to validate before merging a PR.

Reply:
Actually the "api-usage" rules are not "recording rules" and the apiusage.yaml file is not suitably formatted by promtool requirement, but I extracted the rules into separate file test.yaml and verified they are well-formed (which is expected because they are copied AS-IS from cluster-kube-apiserver-operator)
```
michael@BeQuiet:~/hypershift/my-hypershift-fork$ make promtool
cd ./hack/tools; GO111MODULE=on GOFLAGS=-mod=vendor go run github.com/prometheus/prometheus/cmd/promtool check rules ../../cmd/install/assets/recordingrules/*.yaml
Checking ../../cmd/install/assets/recordingrules/hypershift.yaml
  SUCCESS: 5 rules found

Checking ../../cmd/install/assets/recordingrules/test.yaml
  SUCCESS: 2 rules found
```

Reply:
can you add a comment referencing the canonical source?